bucket/hekasoft-backup-restore.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.95",
+    "description": "Hekasoft Backup and Restore is a free tool to backup and restores your browser settings. It allows you to migrate your profile from one browser to another and even custom your profile by removing unwanted files",
+    "homepage": "http://www.hekasoft.com/hekasoft-backup-restore/",
+    "license": "Freeware",
+    "url": "https://www.fosshub.com/Hekasoft-Backup-Restore.html/backup-restore-0.95-portable.zip",
+    "hash": "cd85023db4fdc0c16d4edd2722418978ca3ace855edf52bb2f8660d9a330d561",
+    "extract_dir": "hbr095",
+    "shortcuts": [
+        [
+            "hbr.exe",
+            "Hekasoft Backup and Restore"
+        ]
+    ],
+    "checkver": {
+        "url": "https://www.fosshub.com/Hekasoft-Backup-Restore.html",
+        "regex": "backup-restore-(\\d+(?:\\.\\d+)*)-portable\\.zip"
```

Comment:
I think `backup-restore-([\\d.]+)-portable\\.zip` is enough.
Is there any particular reason to use non-capturing groups (`?:`) ?